.github/workflows/approve-bot-pr.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           name: "event-payload"
           repo: ${{ github.repository }}
           run_id: ${{ github.event.workflow_run.id }}
-          workspace: ${{ github.workspace }}
+          workspace: ${{ github.event_path }}
```

Comment:
Based on the [documentation of the `github` context](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context), and my previous experience having used this property, I don't think this will work. My understanding is that this will return a path to a file like `/github/workspace/event.json`. You then have [code in your action](https://github.com/paketo-buildpacks/github-config/blob/approve-bot-prs/actions/pull-request/download-artifact/entrypoint/main.go#L65-L69) that wants to write the zip file contents into a path that would end up being `/github/workspace/event.json/event.json`. I don't think this is what you want.

Reply:
I'm confusing myself. Do you think setting it to `/github/workspace` would be preferable since that's what it gets mounted to via `-v "/home/runner/work/mri/mri":"/github/workspace"`?